client/pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.camel-tooling.netbeans</groupId>
+    <artifactId>camel-lsp-client-netbeans</artifactId>
+    <version>0.1.0</version>
+    <packaging>nbm</packaging>
+    <name>Language Support for Apache Camel</name>
+    <description>Provides completion, validation and documentation features for Apache Camel URI elements.</description>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <camel-lsp-server-version>1.14.0</camel-lsp-server-version>
```

Comment:
there is the 1.17.0 which has been released https://github.com/camel-tooling/camel-language-server/tags